.travis.yml

```diff
@@ -42,7 +42,7 @@ matrix:
 before_install:
   # Install miniconda and create TEST env.
   - |
-    wget http://bit.ly/miniconda -O miniconda.sh
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
```

Comment:
That `bit.ly` URL is stale. Up-to-date examples on how to set CIs up can be found in https://github.com/ioos/ioos-python-package-skeleton/blob/master/.travis.yml#L26